src/components/store.js

```diff
@@ -1,5 +1,5 @@
 import { createStore } from 'redux';
-import reducer from './reducers';
+import reducer from '../components/reducers/BaseReducer';
```

Comment:
I think you're hooking this up to the wrong reducer, it should be going to your actionReducer. I'd pull actionReducer and your test file into this folder and delete baseReducer.